src/main/java/org/jenkinsci/gradle/plugins/jpi2/V2JpiPlugin.java

```diff
@@ -113,13 +113,22 @@ public void execute(@NotNull Configuration c) {
                     @Override
                     public void execute(@NotNull DependencySet dependencies) {
                         dependencies.add(project.getDependencies()
-                                .create("org.jenkins-ci.main:jenkins-war:" + project.getProperties().get(JENKINS_VERSION_PROPERTY)));
+                                .create("org.jenkins-ci.main:jenkins-war:" + getJenkinsVersion(project)));
                     }
                 });
             }
         });
     }
 
+    private static String getJenkinsVersion(@NotNull Project project) {
+        Map<String, ?> projectProperties = project.getProperties();
+        if (projectProperties.containsKey(JENKINS_VERSION_PROPERTY)) {
+            return projectProperties.get(JENKINS_VERSION_PROPERTY).toString();
+        } else {
+            return "latest.release";
```

Comment:
Is this definitely what we want here? It will resolve to a weekly most of the time

Reply:
Setting it to 2.492.3 - latest LTS right now.